modules/ROOT/pages/sending-data-from-arm-to-external-analytics-software.adoc

```diff
@@ -158,321 +102,259 @@ INDEXED_EXTRACTIONS = JSON
+a|Splunk index name to which to send all events
 
-[NOTE]
-This feature requires the Runtime Manager agent version 1.3.1 or later.
+If the index doesn't exist, the internal handler creates it if the user has permission.
+|main
 
-. First you must obtain a token from Splunk. To do so:
-.. Sign in to your Splunk account.
-.. Navigate to *Settings* -> *Data Inputs*.
-.. Among the different options, you can find the *HTTP Event Collector*, click the *Add New* link next to it.
-+
-image::splunk-datainput-setup.png[Splunk settings]
-.. Follow the steps of the wizard to set up a data input and obtain the token for it
+|Splunk Source
+|The source used for events sent to Splunk
+|mule-events 
 
-. Back in the Runtime Manager, select the server whose information you want to send out.
-. In the menu on the right, click *Manage Server* to access the server's settings.
-+
-image::sending-data-from-arm-to-external-monitoring-software-manage-server.png[]
+|Splunk Type 
+|The sourcetype used for events sent to Splunk
+|mule
 
-. Select the *Plugins* tab:
-+
-image::sending-data-from-arm-to-external-monitoring-software-plugins.png[]
+|Date Format
+|Date format to use to format the timestamp
+|yyyy-MM-dd'T'HH:mm:ssSZ
+|===
 
-. Select the kind of information that you want to send out in the *Level* dropdown menu.
-+
-image::sending-data-from-arm-to-external-monitoring-software-level.png[]
 
-. On the *Event Tracking* region, activate the *Splunk* switch. This will open a pop up menu where you can provide your Splunk user and password data, as well as the host and port for the connection.
-+
-image::sending-data-from-arm-to-external-monitoring-software-splunk.png[]
+// .Splunk Internal Handler Minimum Configuration Example
+// [source,yaml]
+// ----
+//   mule.agent.gw.http.handler.splunk:
+//     host: 192.168.61.131
+//     user: admin
+//     pass: test
+// ----
 
-+
-image::agent-to-splunk-httpevent.png[splunk]
 
-. Select the *HTTP Event Collector* option and then paste the token that Splunk gave you.
-. Optionally, you can open the advanced menu and set up certain formatting properties of the data that will be sent out.
-+
-image::agent-to-splunk-httpevent-advanced.png[splunk]
+== Integrating with Your ELK Stack
 
-[NOTE]
-Although you can set values for the Splunk index, Splunk source and Splunk source type when registering your data input in your Splunk account, the values are overwritten by the values you configure for these fields in the *Advanced* section of the Runtime Manager agent *Plugins* menu.
+The ELK (Elasticsearch, Logstash, Kibana) stack enables you to store, search, and analyze the log data.
 
-=== TCP
 
-This feature require 1.3.1 agent version or later.
+You can configure the Runtime Manager agent to send event notifications from Mule runtime flows to a configurable log file.
+Logstash captures and indexes the data.
+Then, you can use Elasticsearch and Kibana to generate graphs, reports, alerts, dashboards, and visualizations.
```

Comment:
Just an idea:

Logstash first captures and indexes the data; then, you can use Elasticsearch and Kibana to generate graphs, reports, alerts, dashboards, and visualizations